library/crates/Cargo.toml

```diff
@@ -6,26 +6,26 @@ version = "0.0.0"
 path = "" # ignored by cargo raze
 
 [dependencies]
-antlr-rust = "0.3.0-beta"
-axum = "0.5.15"
-chrono = "0.4.22"
-cxx = "1.0.59"
-derivative = "2.2.0"
-enum_dispatch = "0.3.8"
-futures = { version = "0.3.21", features = ["executor", "thread-pool"] }
-grpc = "0.8.3"
-grpc-compiler = "0.8.3"
-grpc-protobuf = "0.8.3"
-log = "0.4.6"
-protobuf = { version = "2.18.2", features = ["with-bytes"] }
-protobuf-codegen = "2.18.2"
-protoc-rust-grpc = "0.8.3"
-rocksdb = "0.17.0"
-tokio = { version = "1.20.1", features = ["full"] }
-uuid = { version = "1.1.2", features = ["fast-rng", "v4"] }
-tonic = "0.8.0"
-prost = "0.11.0"
-tonic-build = "0.8.0"
+antlr-rust = "=0.3.0-beta"
+axum = "=0.5.15"
+chrono = "=0.4.22"
+cxx = "=1.0.59"
+derivative = "=2.2.0"
+enum_dispatch = "=0.3.8"
+futures = { version = "=0.3.21", features = ["executor", "thread-pool"] }
+grpc = "=0.8.3"
+grpc-compiler = "=0.8.3"
+grpc-protobuf = "=0.8.3"
+log = "=0.4.8"
```

Comment:
We had to bump `log` to `0.4.8` because another crate required at least this version.